app/assets/javascripts/discourse/app/templates/user.hbs

```diff
@@ -179,6 +179,7 @@
             {{plugin-outlet name="user-profile-primary" tagName="span" connectorTagName="div" args=(hash model=model)}}
           </div>
         </div>
+        {{plugin-outlet name="user-profile-above-collapsed-info" connectorTagName="div" args=(hash model=model collapsedInfo=collapsedInfo)}}
```

Comment:
Are we 100% sure want it to be a `<div>` always? It looks like neighbour elements are, but we should default to tagless if there's any change people will want to do something different.